### PR TITLE
Support - Update coin coverage scripts to use html-spa reporter & `collectCoverageFrom` + turbo concurrency

### PR DESCRIPTION
**Pull request by lambertkevin**
### ✅ Checklist

- [ ] `npx changeset` was attached. // Uncessary
- [ ] **Covered by automatic tests.**   // Uncessary
- [x] **Impact of the changes:**  
  - no impact

### 📝 Description

- Add a max concurrency for turbo of 2 tests
- Remove `bridge.integration.test.ts` from coverage
- Add blank test when necessary when no test at all
- Add missing script on new coin modules

Capture d’écran 2024-05-17 à 21 52 06

### ❓ Context

- **JIRA or GitHub link**: 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: libs/coin-modules/coin-algorand/package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
   "scripts": {
     "clean": "rimraf lib lib-es",
     "build": "tsc && tsc -m ES6 --outDir lib-es",
-    "coverage": "jest --coverage=true && mv coverage/coverage-final.json coverage/coverage-algorand.json",
+    "coverage": "jest --coverage --collectCoverageFrom='src/**/*.{ts,tsx}' --testPathIgnorePatterns='/bridge.integration.test.ts|node_modules|lib-es|lib/' --passWithNoTests && mv coverage/coverage-final.json coverage/coverage-algorand.json",
```
Comment (sprohaszka-ledger) [inline libs/coin-modules/coin-algorand/package.json:75]: `--collectCoverageFrom='src/**/*.{ts,tsx}'` this line should be unnecessary as `jest.config.js` file already define it (and there is no *tsx* file).

**Review by sprohaszka-ledger — DISMISSED** [review]
Just one thing for ease of maintenance.

File: libs/coin-modules/coin-algorand/package.json (hunk 1 above)
Comment (lambertkevin) [reply libs/coin-modules/coin-algorand/package.json:75]: Alright, removing it 👌

**Review by sprohaszka-ledger — APPROVED** [review]
👍

**Pull request merged**